steps/cadence-pegasus-lvs/run.sh

```diff
@@ -0,0 +1,32 @@
+if [ ! -z "$lvs_hcells_file" ]; then hcell="-hcell $lvs_hcells_file"; fi
+black_box=""
+if [ -e inputs/rules.svrf ]; then
+  while read line; do
+    if [[ $line == *"LVS BOX"* ]]; then
+      arr=($(echo $line | xargs))
+      black_box="${black_box}lvs_black_box ${arr[-1]} -source_layout;\n"
+    fi
+  done < inputs/rules.svrf
+fi
```

Comment:
What's going on here? Are you trying to basically convert the Calibre LVS BOX commands to something Pegasus can support?

Reply:
Yes, that's right